project/Checkstyle.scala

```diff
@@ -100,4 +101,8 @@ object Checkstyle {
     }
   }
 
+  // Enforce java code style
+  lazy val javafmtCheckSettings = Seq(
```

Comment:
this isn't really related to checkstyle itself, so I'm a bit undecided whether this should be moved here